index.html

```diff
@@ -0,0 +1,200 @@
+                    I'm Usama
+                </h1>
+            </div>
+
+            <div class="profession"><h2>I'am a Software Developer</h2></div>
+
+            <div class="containe">
+                <div class="containe1">
+                    <div>
+                        <article class="services">
+                            <p>
+                                I can help you build a product , feature or <br>
+                                 website Look through some of my work <br>
+                                  and experience! If you like what you see <br>
+                                 and have a project you need coded, <br>
+                                don’t hestiate to contact me. <br>
+                            </p>
+                        </article>
+                        <ul class="logo">
+                            <li><a href="https://github.com/UsamaMasood12"><img src="./image/G.png" alt=""></a></li>
+                            <li><a href="www.linkedin.com/in/usama-masood-b4a35014b"><img src="./image/Group.png" alt=""></a></li>
+                            <li><a href="#"><img src="./image/M.png" alt=""></a></li>
+                            <li><a href="https://twitter.com/Usama____Masood"><img src="./image/twitter.png" alt=""></a></li>
+                            <li><a href="#"><img src="./image/V.png" alt=""></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="containe2">
+                    <div class="box box1"><img src="./image/44.png" alt=""></div>
+                </div>
+            </div>
+          </section>
+        <section class="work">
+         <div class="container">
+         <div class="work1"><h1>My Recent Works</h1></div>
+         <div class="vector"><img src="./image/Vector.png" alt=""></div>
+         <div class="Img"><img src="./image/Img Placeholder.png" alt=""></div>
+         <div class="Multi-Post"><h1>Multi-Post Stories</h1></div>
+         <div class="para"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p></div>
+         <ul class="a">
+            <li class="item1">css</li>
+            <li class="item2">html</li>
+            <li class="item3">bootstrap</li>  
+            <li class="item2">Ruby</li>  
+         </ul>
+          <button class="button">See Projects</button>
+        </div>
+        <div class="container2">
+            <div class="Data"><h1>Profesional Art Printing Data</h1></div>
+         <div class="para1"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p></div>
+         <div class="container3">
+            <div class="item4">css</div>
+            <div class="item5">bootstrap</div>
+            <div class="item6">Ruby</div>
+         </div>
```

Comment:
- [ ] As stated by the previous reviewer kindly use `<ul>` and `<li>` tags to group and list your technologies 🙏